src/topp/FileMerger.cpp

```diff
@@ -204,6 +257,12 @@ class TOPPFileMerger :
             it->setMetaValue("file_origin", DataValue(file_list[i]));
           }
         }
+
+        if (rt_gap_ > 0.0) // concatenate in RT
+        {
+          adjustRetentionTimes_(map, trafo_out[i], i == 0);
```

Comment:
i == 0. Seems to be your thing right now ;)
Would a descriptive bool hurt a lot?